internal/app/issue_repository/issue_repository_handler_events.go

```diff
@@ -48,3 +50,50 @@ type DeleteIssueRepositoryEvent struct {
 func (e *DeleteIssueRepositoryEvent) Name() event.EventName {
 	return DeleteIssueRepositoryEventName
 }
+
+// OnIssueRepositoryCreate is a handler for the CreateIssueRepositoryEvent
+// Is adding the default priority  for the default issue repository
+func OnIssueRepositoryCreate(db database.Database, e event.Event) {
+	// TODO: make this configureable
+	var defaultPrio int64 = 100
+	var defaultRepoName string = "nvd"
```

Comment:
This is not required for IssueRepositoryCreate, right?

Reply:
You're right. I've removed it.